src/createFactory.js

```diff
@@ -0,0 +1,43 @@
+export function createFactory(blockFactory, styles, configSchema) {
```

Comment:
Why is this provided as a unique file for every block? Shouldn't we provide it as part of the block scripts module or a new one? We don't want people to touch this, or have to update every block if a change occurs to the factory function.

Reply:
> We don't want people to touch this, or have to update every block if a change occurs to the factory function.

I agree. I left it here for now because I'm not sure where it _should_ live.

Options:
 - Status quo: leave it in the block. Has the issues you described in your comment.
- Export it from the `block-scripts package.
- Export it from the `element-sdk` package.
- Create a new package to export it from. Could be a standalone package for this function only, or made as an SDK/utility package specifically for block development.

Reply:
As much as I don't want to do it, I think moving it to a new package module is the best option here.

Reply:
`createFactory()` is now exported from [`@volusion/element-block-utils`](https://github.com/Volusion/element-block-utils)